lib/daru/index.rb

```diff
@@ -343,7 +345,7 @@ def values
     end
 
     def inspect
-      "Daru::MultiIndex:#{object_id} (levels: #{levels}\nlabels: #{labels})"
+      "#<Daru::MultiIndex(#{levels.map { |l| l.map(&:inspect).join(',') }.join(' > ')})>"
```

Comment:
OK, changing it back, just adding `#<` and `>` around. And maybe line break in inspect is not necessary?..